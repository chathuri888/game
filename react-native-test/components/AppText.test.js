import * as React from "react";
import AppText from "./AppText";
import renderer from "react-test-renderer";
it(`renders correctly`, () => {
    const tree = renderer.create(<AppText>Reset</AppText>);
    expect(tree).toMatchSnapshot();
});