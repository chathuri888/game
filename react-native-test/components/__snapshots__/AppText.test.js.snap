// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#545454",
        "fontFamily": "Avenir",
        "fontSize": 14,
      },
      undefined,
    ]
  }
>
  Reset
</Text>
`;
